src/lib/header-bidding/prebid/prebid.ts

```diff
@@ -225,21 +226,30 @@ class PrebidAdUnit {
 			},
 		};
 
-		this.bids = bids(
-			advert.id,
-			slot.sizes,
-			pageTargeting,
-			this.gpid,
-			consentState,
-		);
+		this.bids = this.bids = shouldIncludePrebidAdUnit
+			? bids(
+					advert.prebidAdUnit,
+					slot.sizes,
+					pageTargeting,
+					this.gpid,
+					consentState,
+				)
+			: bids(
+					advert.id,
+					slot.sizes,
+					pageTargeting,
+					this.gpid,
+					consentState,
+				);
```

Comment:
Bit of a nitpick, feel free to ignore 😬
```suggestion
		this.bids = bids(
					shouldIncludePrebidAdUnit
						? advert.prebidAdUnit
						: advert.id,
					slot.sizes,
					pageTargeting,
					this.gpid,
					consentState,
				);
```